frontend/src/CalendarWeekdayCell/CalendarWeekdayCell.tsx

```diff
@@ -0,0 +1,15 @@
+import classes from './CalendarWeekdayCell.module.css';
+
+type WeekdayCellProps = {
+    day?: string
```

Comment:
probably shouldn't be optional, especially since your component doesn't handle the undefined case as is